@plotly/webpack-dash-dynamic-import/src/index.js

```diff
@@ -10,26 +10,30 @@ function getFingerprint() {
     return `"v${version}m${timestamp}"`;
 }
 
+/**
+ * This code is injected as-is in the webpack artifact and must be ES5 compatible to work in all scenarios.
+ * This will not get transpiled.
```

Comment:
Just to note, because I was curious: it doesn't get transpiled, but it *does* get minified in the pipeline you've got set up - as we would hope 🎉